.gitattributes

```diff
@@ -0,0 +1,8 @@
+* text=auto
+
+/tests              export-ignore
```

Comment:
Personally, I always like to have the tests along with the code to see how it is used etc.
It doesnt hurt to export IMO.